packages/fxa-payments-server/src/lib/Coupon.ts

```diff
@@ -0,0 +1,3 @@
+export type Coupon = {
+  amount: number;
```

Comment:
Could we add the `couponCode`, entered by the user, to this as well? We'll need it in upcoming tickets.